Update section 1 under Setup to address gateway/subgraph versions to use for Fed 1 and Fed 2 contracts (#101)
apollographql/docs: src/content/studio/contracts.mdx

**Before you create any contracts:**

> 1. For Fed 1 contract variants, update your gateway's `@apollo/gateway` library to version 0.34.0 or later. For Fed 2 contract variants, ensure that you are running gateway versions higher than 2.0.2 otherwise you will encounter runtime errors.

Reviewer: ```suggestion
1. For Federation 2 contract variants, ensure that you're running gateway versions higher than 2.0.2. For Federation 1 contract variants, update your gateway's `@apollo/gateway` library to version 0.34.0 or later. Otherwise, you'll encounter runtime errors.
```

> 2. Update your Apollo Server subgraphs to use version 0.1.1 or later of the `@apollo/subgraph` library. For Fed 2 contract variants, ensure that your `@apollo/subgraph` version is higher than 2.0.0. 

Reviewer: ```suggestion
2. For Federation 2 contract variants, update your Apollo Server subgraphs to use version 2.0.0 or later of the `@apollo/subgraph` library. For Federation 1 contract variants, update `@apollo/subgraph` to version 0.1.1 or later.
```
